pytorch_lightning/trainer/trainer.py

```diff
@@ -214,6 +214,10 @@ def __init__(
 
             flush_logs_every_n_steps: How often to flush logs to disk (defaults to every 100 steps).
 
+                .. deprecated:: v1.5
+                    ``flush_logs_every_n_steps`` has been deprecated in v1.5 and will be removed in v1.7.
+                    Please pass ``flush_logs_every_n_steps`` directly to the Logger instead.
```

Comment:
```suggestion
                    Please configure flushing directly in the logger instead.
```
as discussed in thread below, there is no intention to standardize this in the logger api